### ISSUE TITLE
Feature: Editor's size, font-size, font-family

### ISSUE THREAD

**Issue by karap9s** [ozanyurtsever/verbum#34]
**Describe the question**
Is it even possible to set up: 
1. Ediotor's container size.
2. Images max size.
3. Add more font sizes or reduce them.
4. Add my own font families?

**Comment by DiggesT** [comment]
4. Yes, it is possible, use `FontFamilyDropdown` with `fontOptions` parameter, like this:
```
<ToolbarPlugin defaultFontSize="20px">
  <FontFamilyDropdown
    fontOptions={[
      ['Georgia', 'Georgia'],
      ['Times New Roman', 'Times New Roman'],
    ]}
  />
/>
```

**Comment by DiggesT** [comment]
3. Yes, it is possible, just like fonts. Use `FontSizeDropdown` with `fontSizeOptions` parameter:
```
<ToolbarPlugin defaultFontSize="20px">
  <FontSizeDropdown
    fontSizeOptions={[
      ['10px', '10px'],
      ['11px', '11px'],
    ]}
  />
/>
```

**Comment by DiggesT** [comment]
2. Now it's will be available with #48.
 
`<InsertDropdown />` plugin have `enableImage?: { enable: boolean; maxWidth: number };` parameters.
